### PR TITLE
Add condition field to Keycloak log datastream

### PR DESCRIPTION
**Pull request by Alphayeeeet**
Please label as Enhancement

This PR adds the concition field as configurable option to the Keycloak log datastream

### CODE REVIEW

**Comment by Alphayeeeet** [comment]
@andrewkroh FYI this PR is not a bugfix, but rather an enhancement.

File: packages/keycloak/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.23.0"
+  changes:
+    - description: Make condition configurable, when to apply the log datastream
+      type: bugfix
```
Comment (andrewkroh) [inline packages/keycloak/changelog.yml:5]: > FYI this PR is not a bugfix, but rather an enhancement.

Then this should be `enhancement`.

Comment (Alphayeeeet) [reply packages/keycloak/changelog.yml:5]: Ohh my bad, I will fix it tomorrow. Thanks for mentioning

File: packages/keycloak/data_stream/log/manifest.yml [hunk 2]
```diff
@@ -19,6 +19,13 @@ streams:
         show_user: false
         default:
           - keycloak-log
+      - name: condition
+        title: Condition
+        description: Condition to filter when to apply this datastream
```
Comment (andrewkroh) [inline packages/keycloak/data_stream/log/manifest.yml:24]: Is there any precedent in other packages for making this configurable? I am wondering if there is already an established description for this field. To me it's missing context about what is a "condition". I think we need to incorporate a link to the docs (i.e. https://www.elastic.co/guide/en/fleet/current/dynamic-input-configuration.html#conditions).

Comment (Alphayeeeet) [reply packages/keycloak/data_stream/log/manifest.yml:24]: I used prometheus.collector as orientation and copied the description. But if needed, I can still change the description and include the URL.

Comment (Alphayeeeet) [reply packages/keycloak/data_stream/log/manifest.yml]: I now copied a part of teh Kubernetes.container_logs integration, which already contained the link to the docs.

**Comment by Alphayeeeet** [comment]
@andrewkroh Could youe please run CI again?

**Review by andrewkroh — APPROVED** [review]

**Comment by Alphayeeeet** [comment]
@andrewkroh Package tests look alright, but CI has a problem when stopping test environment. Is this an issue of the integration or of the CI environment?

**Comment by andrewkroh** [comment]
It's a CI issue. I am told that https://github.com/elastic/integrations/pull/11227 is testing a fix.

**Pull request merged**